### PR TITLE
Hcap 751 peoi mobile overlap

### PR DESCRIPTION
**Pull request by Zenen-freshworks**
I implemented a drop-down menu without giving much consideration to aesthetics, I'm expecting that this might need some refinement before it goes to production.

### CODE REVIEW

**Review by fwpushan — COMMENTED** [review]
Everything looks good to me. Can you add a link to /participant-landing for BCSC or non employer user?

**Review by fwpushan — APPROVED** [review]
Ignore my previous comment. @dbayly-freshworks already implemented that in subsequent  PR.

File: client/src/components/generic/Header.js [hunk 1]
```diff
@@ -128,43 +140,66 @@ export const Header = ({ hideEmployers = false }) => {
...
+                <KeyboardArrowDownIcon onClick={handleMenuClick} />
+                <Menu
+                  id='login-menu'
+                  anchorEl={anchorEl}
+                  keepMounted
+                  open={Boolean(anchorEl)}
+                  onClose={handleMenuClose}
+                >
+                  {keycloak.authenticated && location.pathname !== Routes.Admin && (
+                    <MenuItem key='Home' onClick={() => history.push(Routes.Admin)}>
```
Comment (dbayly-freshworks) [inline client/src/components/generic/Header.js:187]: In a case where a person is logged in in the participant portal, I think it makes sense to redirect them to the participant landing page. I added some of that function to my last PR, not sure what happened to it.

Comment (Zenen-freshworks) [reply client/src/components/generic/Header.js:187]: I think it got lost in the merge, my bad.  I'll fix it up.

**Review by dbayly-freshworks — APPROVED** [review]

**Review by fwpushan — APPROVED** [review]

**Pull request merged**